docs/data/toolpad/core/components/sign-in-page/sign-in-page.md

```diff
@@ -44,15 +44,37 @@ The following providers are supported and maintained by default:
 - Twitch
 - Discord
 - Keycloak
-- Credentials (username/password)
 
 Find details on how to set up each provider in the [Auth.js documentation](https://authjs.dev/getting-started/authentication/oauth).
 :::
 
+## Magic Link
+
+:::warning
+To use magic links, you must configure a database and pass in the required environment variables to send emails. See more details in the Auth.js docs on [database setup for email](https://authjs.dev/getting-started/authentication/email) and [Nodemailer configuration](https://authjs.dev/getting-started/providers/nodemailer/).
```

Comment:
This page still gives me the impression that Toolpad core is about providing a full integration end-to-end, while in reality it's just about providing UI. I feel like it would make more sense to come from the angle that this component aims to provide UI for the auth.js Nodemailer provider.

Reply:
I can reword the warning and the introduction text of this section like the following:


:::warning
To use magic links, apart from configuring an authentication provider like Auth.js, you must also configure a database and pass in the required environment variables to send emails. See more details in the Auth.js docs on [database setup for email](https://authjs.dev/getting-started/authentication/email) and [Nodemailer configuration](https://authjs.dev/getting-started/providers/nodemailer/).
:::

You can use the `SignInPage` component to quickly setup a sign-in page for the `nodemailer` provider offered by Auth.js which sends one-time verification links to the user's email address.

Reply:
I'd expect something along the lines of

The Signin page component supports magic links. In order to enable this you'll have to set up a provider such as Auth.js NodeMailer. Please note that such provider requires persistence.